liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/InstallFeatureSupport.java

```diff
@@ -159,10 +170,18 @@ else if (util == null && !noFeaturesSection) {
         }
     }
 
-    protected InstallFeatureUtil getInstallFeatureUtil(Set<String> pluginListedEsas) throws PluginExecutionException {
-        InstallFeatureUtil util = null;
+    /**
+     * Get a new instance of InstallFeatureUtil
+     * 
+     * @param pluginListedEsas The list of ESAs specified in the plugin configuration, or null if not specified
+     * @param propertiesList The list of product properties installed with the Open Liberty runtime
+     * @param openLibertyVersion The version of the Open Liberty runtime
+     * @return instance of InstallFeatureUtil
+     */
+    protected InstallFeatureUtil getInstallFeatureUtil(Set<String> pluginListedEsas, List<ProductProperties> propertiesList, String openLibertyVerion) 
+            throws PluginExecutionException {
         try {
-            util = new InstallFeatureMojoUtil(pluginListedEsas);
+            util = new InstallFeatureMojoUtil(pluginListedEsas, propertiesList, openLibertyVerion);
```

Comment:
Now this is both assigning the member variable and also returning it for the caller to assign the member variable.  It's not wrong but kind of strange.  Should do one or the other.